src/wp-includes/class-wp-block.php

```diff
@@ -259,7 +260,28 @@ private function process_block_bindings() {
 			return $computed_attributes;
 		}
 
-		foreach ( $parsed_block['attrs']['metadata']['bindings'] as $attribute_name => $block_binding ) {
+		$bindings = $parsed_block['attrs']['metadata']['bindings'];
+
+		// If the default binding is set for pattern overrides, replace it
+		// with a pattern override binding for all supported attributes.
+		if (
+			isset( $bindings['__default']['source'] ) &&
+			'core/pattern-overrides' === $bindings['__default']['source']
+		) {
+			// Build an binding array of all supported attributes.
+			// Note that this also omits the `__default` attribute from the
+			// resulting array.
```

Comment:
```suggestion
			/*
			 * Build an binding array of all supported attributes.
			 * Note that this also omits the `__default` attribute from the
			 * resulting array.
			 */
```

Reply:
Pushed a commit that updates to use multiline comments 👍